modules/gapi/include/opencv2/gapi/gtyped.hpp

```diff
@@ -39,6 +39,46 @@ namespace detail
     auto make_default()->decltype(T{}) {return {};}
 }; // detail
 
+/**
+ * @brief This class is a typed wrapper over a regular GComputation.
+ *
+ * `std::function<>`-like template parameter specifies the graph
+ *  signature so methods so the object's constructor, methods like
+ *  `apply()` and the derived `GCompiledT::operator()` aslo become
+ *  typed.
+ *
+ *  There is no need to use cv::gin() or cv::gout() modifiers with
+ *  objects of this class.  Instead, all input arguments are followed
```

Comment:
It sounds like a there is no need of using cv::gin(GComputationT())...

Reply:
Not sure if it really does